metadata-ingestion/src/datahub/utilities/urn_encoder.py

```diff
@@ -1,6 +1,8 @@
 import urllib.parse
 from typing import List
 
+# NOTE: Frontend relies on encoding these three characters. Specifically, we decode and encode schema fields for column level lineage.
```

Comment:
Question - In such cases, how will these column fields appear in 

a) Schema Tab
b) Stats Tab

Do we also need that encoding there? If yes, maybe consider adding in this PR, or marking TODOs.

Reply:
great question - I believe these tabs do not need to be decoded, since the tables in them are reading from fields in mysql that have the `fieldPath` stored in them, not from `schemaField` urns that we generate and encode.

So the only time we have encoded fieldPaths is when we're actually grabbing the `fieldPath` from the schema field urn - and we only have those stored in `upstreamLineage`, `inputFields`, and now that you're calling this out, I discovered we're also storing them in `DatasetAssertionInfo` and under `foreignKeys` in `SchemaMetadata` - going to add some coverage for that just in case

Reply:
Nice!